### ISSUE TITLE
Add xdg-toplevel-drag protocol to scope

### ISSUE THREAD

**Issue by ludgie** [labwc/labwc-scope#12]
Though I don't know if this protocol should be implemented as-is to labwc or through wlroots as wlr-protocol?
But Simon Ser said he didn't have any interest on implementing it to wlroots... https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/204#note_1918367

https://gitlab.freedesktop.org/wayland/wayland-protocols/-/tree/main/staging/xdg-toplevel-drag

**Comment by Consolatis** [comment]
> Though I don't know if this protocol should be implemented as-is to labwc or through wlroots as wlr-protocol?

Usually all protocols are implemented through wlroots.

> But Simon Ser said he didn't have any interest on implementing it to wlroots.

To add to that statement: based on the linked post he doesn't personally have an interest in working on it, that doesn't mean that wlroots would reject a MR to implement the protocol.

**Comment by ludgie** [comment]
> To add to that statement: based on the linked post he doesn't personally have an interest in working on it, that doesn't mean that wlroots would reject a MR to implement the protocol.

Yes, I'm aware. Didn't add that part. Sorry

Then should I close this?

**Comment by Consolatis** [comment]
Not necessarily. I don't have a strong opinion about the feature personally but it seems @jlindgren90 would really like it to be implemented at some point.

**Comment by jlindgren90** [comment]
Any application that uses floating panels/toolbars will need xdg-toplevel-drag to work properly under native Wayland. So I think we'll definitely want to support it eventually. For the time being, those applications can be run via XWayland and should be fine.

Examples: Audacious, many Qt/KDE apps that allow detachable toolbars - I think also Chromium for detachable tabs.

**Comment by geerlingguy** [comment]
@jlindgren90 - And as @argilo recently discovered, GQRX :)
